server/src/main/java/com/linecorp/centraldogma/server/internal/api/auth/ProjectOwnersOnly.java

```diff
@@ -14,21 +14,22 @@
  * under the License.
  */
 
-package com.linecorp.centraldogma.server.internal.admin.decorator;
-
-import javax.annotation.Nullable;
+package com.linecorp.centraldogma.server.internal.api.auth;
 
+import com.linecorp.armeria.common.HttpRequest;
+import com.linecorp.armeria.server.ServiceRequestContext;
 import com.linecorp.centraldogma.server.internal.admin.authentication.User;
-import com.linecorp.centraldogma.server.internal.admin.model.ProjectRole;
+import com.linecorp.centraldogma.server.internal.metadata.ProjectRole;
 
 /**
  * A decorator for access control. It only allows a request from an administrator and a user who has
  * a {@link ProjectRole#OWNER} role.
  */
-public class ProjectOwnersOnly extends ProjectAccessController {
+public class ProjectOwnersOnly extends AbstractRoleCheckingDecorator {
 
     @Override
-    protected boolean isAllowedRole(User user, @Nullable ProjectRole projectRole) {
-        return user == User.ADMIN || projectRole == ProjectRole.OWNER;
+    protected boolean isAccessAllowed(ServiceRequestContext ctx, HttpRequest req,
+                                      User user, ProjectRole role) {
+        return user.roles() == User.LEVEL_ADMIN || role == ProjectRole.OWNER;
```

Comment:
`User.roles()` returns `List<String>`, which means it will never be same with `User.LEVEL_ADMIN`?